app/controllers/contact_embed_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class ContactEmbedController < ApplicationController
+  rescue_from GdsApi::BaseError do |e|
+    GovukError.notify(e)
+    render "new_api_down", layout: rendering_context
```

Comment:
I think this name may be confusing. _Is it using the new API?_

Reply:
I hadn't thought of it like that! This is derived from the 'new' action, and we've [got some precedent for this naming](https://github.com/alphagov/content-publisher/blob/master/app/controllers/access_limit_controller.rb#L6) throughout the app.